Formula/dssp.rb

```diff
@@ -0,0 +1,44 @@
+class Dssp < Formula
+  # cite Touw_2015: "https://doi.org/10.1093/nar/gku1028"
+  # cite Kabsch_1983: "https://doi.org/10.1002/bip.360221211"
+  desc "Create DSSP files"
+  homepage "https://github.com/cmbi/dssp"
+  url "https://github.com/cmbi/dssp/archive/3.1.4.tar.gz"
+  sha256 "496282b4b5defc55d111190ab9f1b615a9574a2f090e7cf5444521c747b272d4"
+
+  bottle do
+    root_url "https://linuxbrew.bintray.com/bottles-bio"
+    cellar :any
+    sha256 "c0833e3e3db899b28dd9515e1d3ffb791528d2934d25226dbf4eaf0f0428b34f" => :sierra
+    sha256 "890f4a0285987df828d271ad333b597181358c75b65026d94dccf3d7c6d49531" => :x86_64_linux
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "boost"
```

Comment:
Is there run-time linkage to `boost`? What's the output of `brew linkage dssp`?

Reply:
Here is the output of `brew linkage dssp`.

```
$ brew linkage dssp
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
Homebrew libraries:
  /usr/local/opt/boost/lib/libboost_date_time-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_filesystem.dylib (boost)
  /usr/local/opt/boost/lib/libboost_iostreams-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_program_options-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_system.dylib (boost)
  /usr/local/opt/boost/lib/libboost_thread-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_unit_test_framework-mt.dylib (boost)
```

But another issue has happened on the build-linux-bottles.